bubbleSort/bubbleSort.swift

```diff
@@ -0,0 +1,14 @@
+    func bubbleSort(_ array: [Int]) -> [Int] {
```

Comment:
```suggestion
func bubbleSort(_ array: [Int]) -> [Int] {
```
Nice one, but could you please update it, to not have empty tabs there